src/app/model/jit/transport-call.ts

```diff
@@ -29,4 +29,5 @@ export interface TransportCall {
   atdBerthDateTime?: Date;
   latestEventCreatedDateTime?: Date;
   transportCallID: string;
+  milesRemainingToDestination?: number;
```

Comment:
To be renamd to `milesToDestinationPort`

Reply:
same as above